### ISSUE TITLE
What is the status of this project?

### ISSUE THREAD

**Issue by JaneJeon** [ClearURLs/Addon#144]
First of all, thanks for the project! I'd like to inquire about the status of this project.

There hasn't been an update of the rules json in almost a year and there do seem to be lots of issues (in both this repo and the rules repo) about links breaking or trackers not being stripped off that just sit untouched since January.

The reason I ask this is because I maintain a downstream project that relies on clearURLs called https://github.com/JaneJeon/normalize-url-plus and it uses clearURLs because AFAIK it is (so far) the best list of URL trackers.

I understand that the _extension_ is undergoing a rewrite, but it would be nice to know if the _rules_ at the very least are still being actively being maintained/fixed/updated, or if it's not, whether it is planned to be picked up anytime soon (if so, what's the timeline?), whether the rules are abandoned, whether the author(s) are intending to keep the rules up to date but just need some help, etc.

Thanks!

@KevinRoebert 

**TL;DR:** even if the extension is undergoing a rewrite or is dropped outright, there is enough value in frequently updating and actively maintaining the tracker rule list *independently* from the rewrite, and I think there would be enough people willing to do that if you are too short on time/need some help.

**Comment by JaneJeon** [comment]
Related: #131 
#126

**Comment by JaneJeon** [comment]
Also a quick pimp: a LOT of the "oh my god this tracker doesn't get stripped out!" can be resolved (actually, I have been able to successfully normalize/clean all of the URLs for ALL of the current open issues *except* for tiktok, which does need custom rules on clearURLs side) by using normalize-url-plus which does link following + canonical link extraction.

So the ruleset I think could be slimmed down a lot by integrating just link following + canonical extraction, or just using normalize-url-plus directly.

**Comment by KoalaBear84** [comment]
Yes. It would be nice if there was an update to the projects status. As it looks like it is completely inactive which is sad after such a good start and cause.

**Comment by JaneJeon** [comment]
@KoalaBear84 for what it's worth, the GitLab one seems to be unmaintained as well. I also have opened an issue on the gitlab repo

**Comment by KoalaBear84** [comment]
Yes, I've seen there is also not a lot of activity on there indeed, at least some.

**Comment by KevinRoebert** [comment]
> First of all, thanks for the project! I'd like to inquire about the status of this project.
> 
> There hasn't been an update of the rules json in almost a year and there do seem to be lots of issues (in both this repo and the rules repo) about links breaking or trackers not being stripped off that just sit untouched since January.
> 
> The reason I ask this is because I maintain a downstream project that relies on clearURLs called https://github.com/JaneJeon/normalize-url-plus and it uses clearURLs because AFAIK it is (so far) the best list of URL trackers.
> 
> I understand that the _extension_ is undergoing a rewrite, but it would be nice to know if the _rules_ at the very least are still being actively being maintained/fixed/updated, or if it's not, whether it is planned to be picked up anytime soon (if so, what's the timeline?), whether the rules are abandoned, whether the author(s) are intending to keep the rules up to date but just need some help, etc.
> 
> Thanks!
> 
> @KevinRoebert
> 
> **TL;DR:** even if the extension is undergoing a rewrite or is dropped outright, there is enough value in frequently updating and actively maintaining the tracker rule list _independently_ from the rewrite, and I think there would be enough people willing to do that if you are too short on time/need some help.

Hi. Yes, the project is still maintained by me. As already mentioned in other issues, I don't have so much time at the moment. But from February 2022, I have much more time again. At the latest, then I will add the whitelist feature to ClearURLs.

The rules will, of course, be maintained as well. Anyone who wants to help can take an issue and create a merge request for it in the [rules repo](https://gitlab.com/anti-tracking/ClearURLs/rules/). I will try to review the MR as soon as possible and merge it into the master.

I am happy about any help, especially with maintaining the already slightly expanded rules catalog.

**Comment by corobin** [comment]
>  using normalize-url-plus which does link following + canonical link extraction.

doesn't that actually hit the link? so the destination gets to see the tracking link anyway
